package/Windows/Actions/CustomAction.cpp

```diff
@@ -831,7 +941,7 @@ UINT __stdcall ValidateCertificate(MSIHANDLE hInstall)
 
 		for (int i = 0; i < lstrlenW(szCertPass); i++)
 		{
-			szValBuf[i] = L'�';
+			szValBuf[i] = L'�';
```

Comment:
this comes off as 'ï¿½' in GitHub - I just assume it's encoded the right way in the source file?

Reply:
It doesn't even show in the local diff. I will double-check my editor didn't mangle this before merging.